src/app/public/modules/modal/modal.interface.ts

```diff
@@ -26,27 +26,30 @@ export interface SkyModalConfigurationInterface {
   providers?: any[];
 
   /**
-   * Sets the `aria-describedby` attribute for the modal dialog to support accessibility.
-   * The value should be an ID (without the leading `#`) that points to the element
-   * that describes the modal. Typically, this is the text on the modal,
-   * not including anything that users interact with such as buttons or a form.
-   * If no ID is specified, the default value is the content of the `sky-modal-content` component.
+   * Specifies the HTML element ID (without the leading `#`) of the element that describes
+   * the modal. This sets the modal's `aria-describedby` attribute
+   * [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
+   * The description typically includes text on the modal but not on items that users interact
+   * with, such as buttons and forms. If you do not specify an ID, the default description is
+   * the content of the `sky-modal-content` component.
    */
   ariaDescribedBy?: string;
 
   /**
-   * Sets the `aria-labelledby` attribute for the modal dialog to support accessibility.
-   * The value should be an ID (without the leading `#`) that points to the element
-   * that labels the modal. Typically this is a header element. If no ID is specified,
-   * the default value is the content of the `sky-modal-header` component.
+   * Specifies the HTML element ID (without the leading `#`) of the element that labels
+   * the modal. This sets the `aria-labelledby` attribute for the modal
+   * [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
+   * This is typically a header element, and if you do not specify an ID, the default value
+   * is the content of the `sky-modal-header` component.
    */
   ariaLabelledBy?: string;
 
   /**
-   * Specifies [an ARIA role](http://www.w3.org/WAI/PF/aria/roles) for the modal dialog
-   * to support accessibility by indicating how the modal functions and what it controls.
-   * The ARIA role indicates what the modal component represents on the web page.
-   * This property defaults to `dialog`.
+   * Specifies an ARIA role for the modal
+   * [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility)
+   * by indicating how the modal functions and what it controls. For information about
+   * how an ARIA role indicates what an item represents on a web page, see the
+   * [WAI-ARIA roles model](http://www.w3.org/WAI/PF/aria/roles).
```

Comment:
We're missing the default:
`This property defaults to dialog.`

Reply:
I tried adding the default, but it doesn't show up when I serve the docs. I added it just like I've seen defaults added elsewhere:
![image](https://user-images.githubusercontent.com/7850105/101174905-78448600-3612-11eb-939f-90741105470c.png)
but it doesn't show up in the docs:
![image](https://user-images.githubusercontent.com/7850105/101174925-81cdee00-3612-11eb-9f06-97dc445e4b1b.png)
Any idea what I'm doing wrong here? Or is this possibly a `docs-tools` issue?

Reply:
Ah, that's because this is an interface. Interface's don't really have default values. We may be able to describe what the default value is elsewhere, but its such low priority, I think its ok for now. I'll approve it.